VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss

```diff
@@ -41,7 +41,7 @@
 
 .vh-name {
   display: inline-block;
-  width: 210px;
+  width: 195px;
```

Comment:
can we move away from harcoded widths and make this panel flexible?

Reply:
updated to percent